biopet/biopet.wdl

```diff
@@ -264,10 +264,29 @@ task ScatterRegions {
           ~{"-L " + regions} \
           ~{"--bamFile " + bamFile} \
           ~{true="--notSplitContigs" false="" notSplitContigs}
-    }
+
+        # Glob messes with order of scatters (10 comes before 1), which causes
+        # problems at gatherGvcfs
+        # Therefore we reorder the scatters with python.
+        # Copy all the scatter files to the CWD so the output matches paths in
+        # the cwd.
+        for file in ~{outputDirPath}/*
+          do cp $file .
+        done
```

Comment:
Is this for loop needed? I'm not really seeing its purpose.

Reply:
Same purpose as in the reorderedGlobbedScatters function.
All the scatters are in `[scatters/scatter-x.bed, ...]` but `os.listdir("scatters") gives `[scatter-x.bed, ...]` . So this copy prevents some annoying stuff.

Reply:
Oh right, now they'll be taken from CWD instead of scatters. I got confused since `os.listdir` still used the `outputDirPath`.

Would it be better to turn the print into something like `print("{}/{}".format("~{outputDirPath}, x))`? Than we won't need to make copies.

Reply:
Much better :+1:. On it.